web/cobrands/hampshire/assets.js

```diff
@@ -0,0 +1,75 @@
+(function(){
+
+if (!fixmystreet.maps) {
+    return;
+}
+
+var wfs_host = fixmystreet.staging ? 'tilma.staging.mysociety.org' : 'tilma.mysociety.org';
+var tilma_url = "https://" + wfs_host + "/mapserver/hampshire";
+
+var defaults = {
+    http_options: {
+        url: tilma_url,
+        params: {
+            SERVICE: "WFS",
+            VERSION: "1.1.0",
+            REQUEST: "GetFeature",
+            SRSNAME: "urn:ogc:def:crs:EPSG::27700"
+        }
+    },
+    geometryName: 'msGeometry',
+    srsName: "EPSG:27700",
+    body: "Hampshire County Council",
+    strategy_class: OpenLayers.Strategy.FixMyStreet
+};
+
+var highways_style = new OpenLayers.Style({
+    fill: false,
+    strokeColor: "#5555FF",
+    strokeOpacity: 0.1,
+    strokeWidth: 7
+});
+
+fixmystreet.assets.add(defaults, {
+    http_options: {
+        params: {
+            TYPENAME: "Road_Sections"
+        }
+    },
+    stylemap: new OpenLayers.StyleMap({
+         'default': highways_style
+    }),
+    usrn: {
+        attribute: 'SITE_CODE',
+        field: 'site_code'
+    },
+    road: true,
+    asset_item: 'road',
+    asset_type: 'road',
+    no_asset_msg_id: '#js-not-a-road',
+    always_visible: true,
```

Comment:
Why does this layer have both always_visible and an asset_group? They seem contradictory - and mean it's impossible to report anything not on a road in any category.